release/Makefile

```
@@ -30,7 +30,14 @@ $(OUTDIR)/opam-full-$(VERSION).tar.gz:
 	mkdir -p "$(OUTDIR)"
 	git clone $(GIT_URL) -b $(TAG) "$(OUTDIR)/opam-full-$(VERSION)"
 	$(MAKE) -C "$(OUTDIR)/opam-full-$(VERSION)" OCAML=$(call pathsearch,ocaml) download-ext
-	cd "$(OUTDIR)" && tar cz -f $(notdir $@) --exclude .git opam-full-$(VERSION)
+	( set -euo pipefail && \
+	  cd "$(OUTDIR)/opam-full-$(VERSION)" && \
+	  git archive "$(TAG)" \
+		--prefix "opam-full-$(VERSION)/" \
+		--mtime "$$(git show -s --format=%ct "$(TAG)" | tail -1)" \
+		-o "$(abspath $(OUTDIR)/opam-full-$(VERSION).tar)" \
+		$$(for f in $$(git ls-files -i -o -x '*'); do echo "--add-file=$$f" || exit 1; done) && \
```

Doing it directly in make has the benefit that the displayed command explicitly shows what was added (I haven’t run it - I’m 99.9% sure the asterisk doesn’t want quoting when used with `$(shell`!)
```suggestion
		$(addprefix --add-file=,$(shell git ls-files -i -o -x *)) && \
```